### DISCUSSION TITLE
Using Site Column of type Choice in a search filter?

### DISCUSSION THREAD

**Discussion by tharrison01** [microsoft-search/pnp-modern-search#2968] [Q&A]
I have created a custom Content Type in SharePoint and added some Site Columns, one is a Choice type and I have added some entries in my test documents.
There is a mapping between a Managed Property and the Crawled property but when I add this to the search I see the multiple choices as a delimited string.
Can those choices somehow be presented as separate options in the filter?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/133253349/fb7b86b2-7cfd-425b-8c56-cb47500f613d)

**Comment by kasperbolarsen** [comment]
The Choice site column seems to be multi selection,  which makes it unusable as a refiner.

In generel multi select is an issue when using as a refiner, so hopefully you can change the field to single select.
